### PR TITLE
Update release WDL to the new monorepo Utopia.

### PR DESCRIPTION
**Pull request by mcovarr**
Tested this out on a fork over here: https://github.com/mcovarr/cromwell

### CODE REVIEW

File: release/release_workflow.wdl [hunk 1]
```diff
@@ -121,73 +68,22 @@ task do_release {
...
-
 task versionPrep {
     String organization
-    String repo
-    String file
-    String regexPrefix
     String updateCommandTemplate
-    
-    String bash_rematch = "{BASH_REMATCH[1]}"
+    String repo = "cromwell"
```
Comment (cjllanwarne) [inline release/release_workflow.wdl]: Is `organization` also a fixed value?

File: release/release_workflow.wdl [hunk 2]
```diff
@@ -235,151 +131,33 @@ task makeGithubRelease {
...
-       dependencyName = cromwellDepName,
-       newVersion = cromwellVersionName,
-       dependencyFilePath = "project/Dependencies.scala"
-    }
-  }
-  
-  File cromwellJar = release_cromwell.executionDir + "/target/scala-2.12/cromwell-" + cromwellPrep.version + ".jar"
-  # Version that was just released
-  Int cromwellVersionAsInt = cromwellPrep.version
+  File cromwellJar = do_release.executionDir + "/target/scala-2.12/cromwell-" + versionPrep.currentVersion + ".jar"
```
Comment (cjllanwarne) [inline release/release_workflow.wdl]: Might look nicer with interpolation?

File: release/release_workflow.wdl [hunk 3]
```diff
@@ -235,151 +131,33 @@ task makeGithubRelease {
...
-  
+  Int cromwellPreviousVersionAsInt = cromwellVersionAsInt - 1
+
   call makeGithubRelease { input:
            githubToken = githubToken,
            organization = organization,
            cromwellJar = cromwellJar,
            newVersion = cromwellVersionAsInt,
-           oldVersion = cromwellPreviousVersion
+           oldVersion = cromwellPreviousVersionAsInt
```
Comment (cjllanwarne) [inline release/release_workflow.wdl]: Just OOI, are we losing all separate versioning for subprojects? E.g. `wdl4s-0.15` is now just `cromwell-wdl-30`?

Comment (geoffjentry) [reply release/release_workflow.wdl]: I've been thinking lately that that wouldn't be a bad thing for book keeping purposes. The flip side is that if we don't touch the artist currently known as `wdl4s` that we're revving versions on people and that seems sucky.

File: release/release_workflow.wdl (hunk 1 above)
Comment (mcovarr) [reply release/release_workflow.wdl]: Oh no, that's how this is tested. You clone broadinstitute/cromwell and test in your own organization. 🙂

File: release/release_workflow.wdl (hunk 3 above)
Comment (mcovarr) [reply release/release_workflow.wdl]: Yes all separate versioning was lost in the monorepoing, not in this PR.

File: release/release_workflow.wdl (hunk 1 above)
Comment (cjllanwarne) [reply release/release_workflow.wdl]: Ha, ok then!

File: release/release_workflow.wdl (hunk 2 above)
Comment (mcovarr) [reply release/release_workflow.wdl]: sure

**Pull request merged**